site-book/README.md

```diff
@@ -1,13 +1,14 @@
 # Metron Site-Book documentation
 
-Metron's Site Book is an attempt at producing documentation that is
+Metron's Site Book is an attempt at producing documentation that is:
 
 - Versioned and reviewed
 - Tied to code versions
 - Highly local to the code being documented
 
 The idea is that a release manager would build the site-book (following the instructions below), then publish it from the public [Metron site](http://metron.incubator.apache.org/) as the docs for the new released version. Older site-book versions should remain available for users that need them.
 
+
 To build the book, do the following:
```

Comment:
I meant adding an additional newline at the end of this line.  If you [view this line](https://github.com/ottobackwards/incubator-metron/blob/31b517aeea1e4afb73e45910db523c2660aa8584/site-book/README.md) post-parsing you'll see lines 12 and 13 are both together right now.